internal/querycoord/meta.go

```diff
@@ -599,6 +599,11 @@ func (m *MetaReplica) GetQueryChannel(collectionID UniqueID) (string, string) {
 		QueryChannelID:       allocatedQueryChannel,
 		QueryResultChannelID: allocatedQueryResultChannel,
 	}
+	err := saveQueryChannelInfo(collectionID, queryChannelInfo, m.client)
+	if err != nil {
+		//TODO:: deal err info
+		log.Error("GetQueryChannel: save channel to etcd error", zap.Error(err))
```

Comment:
not return?

Reply:
TODO in pr #7679, ut is hard to add if return err in this pr?

Reply:
There is very little diff in this pr. If err is returned, ut needs to be added to cover the case of etcd write failure.